inspirehep/modules/search/walkers/elasticsearch.py

```diff
@@ -90,6 +90,12 @@ def visit(self, node):
     @visitor(Value)
     def visit(self, node):
         def query(keyword):
+
+            # FIXME: This is a temporary hack that should be removed when
+            # nested keywords search is implemented
```

Comment:
Nit: you can remove the space between the function declaration and the comment. Maybe indent the text and add a dot at the end:
```
# FIXME: This is a temporary hack that should be removed when
#              we implement proper nested keywords search.
```